python/pyspark/errors/README.md

```diff
@@ -0,0 +1,87 @@
+# Guidelines
```

Comment:
Let's add this into dev guidelines https://github.com/apache/spark/blob/master/python/docs/source/development/contributing.rst